render.yaml

```diff
@@ -4,6 +4,12 @@ databases:
     user: hostedgpt
     plan: free
 
+envVarGroups:
+- name: generated-secrets
+  envVars:
+    - key: rails_master_key
+      generateValue: true
+
```

Comment:
Oh, interesting! @robacarp how does this make it so we don’t have to copy that master.key value?

My assumption was: the master.key value is important and the fix would involve telling the rails app to find the key within the master.key file, not within the ENV. But it seems like you’ve fixed it somehow by telling render to generate a new one? Is there a discrepancy between this newly generated value and the value already stored in master.key?